dev-tools/docker/linux_image/Dockerfile

```
@@ -133,10 +133,13 @@ RUN \
   export BLAS=Eigen && \
   export PYTORCH_BUILD_VERSION=1.7.1 && \
   export PYTORCH_BUILD_NUMBER=1 && \
-  cd ${build_dir}/pytorch && \
   /usr/local/bin/python3.7 setup.py install && \
   mkdir /usr/local/gcc93/include/pytorch && \
   cp -r torch/include/* /usr/local/gcc93/include/pytorch/ && \
   cp torch/lib/libtorch_cpu.so /usr/local/gcc93/lib && \
   cp torch/lib/libc10.so /usr/local/gcc93/lib && \
-  rm -rf ${build_dir}/*
+  cd .. && rm -rf ${build_dir}/*
+
+FROM centos:7
+COPY --from=0 /usr/local/gcc93/lib /usr/local/gcc93/lib
+COPY --from=0 /usr/local/gcc93/include /usr/local/gcc93/include
```

You'll need more than this.  The compiler itself is in `/usr/local/gcc93/bin` and `/usr/local/gcc93/lib64`.  It's probably best to just copy the whole `gcc93` directory structure:

```suggestion
COPY --from=0 /usr/local/gcc93 /usr/local/gcc93
```

Same for aarch64.